.github/workflows/full-ci.yml

```diff
@@ -133,6 +133,10 @@ jobs:
   ci_docker:
     uses: ./.github/workflows/build-dockerfile.yml
 
+  ci_rest:
+    uses: ./.github/workflows/ci-rest.yml
+    secrets: inherit
+
   # dummy job for branch protection check
   full_ci_passed:
     needs: [
```

Comment:
should we add `ci_rest` in this list too?

Reply:
Done [fa4ada4](https://github.com/TileDB-Inc/TileDB/pull/5610/commits/fa4ada407b4dd8a6888c6c6c27ee374b670a3de2)